src/chromium/crBrowser.ts

```diff
@@ -52,6 +52,7 @@ export class CRBrowser extends platform.EventEmitter implements Browser {
     const promises = [
       session.send('Target.setDiscoverTargets', { discover: true }),
       session.send('Target.setAutoAttach', { autoAttach: true, waitForDebuggerOnStart: true, flatten: true }),
+      session.send('Target.setDiscoverTargets', { discover: false }),
```

Comment:
Is this temporary before you land your chromium patch?

Reply:
No, it's permanent, we don't need targetCreated/Destroyed notifications anymore, just attachedToTarget/detachedFromTarget.